resources/config.rb

```diff
@@ -2,8 +2,8 @@
 
 attribute :services_file, :kind_of => String, :default => node["kagent"]["services"]
 attribute :role, :kind_of => String, :name_attribute => true, :required => true
-attribute :service, :kind_of => String, :required => true
-attribute :log_file, :kind_of => String 
+attribute :service, :kind_of => String
```

Comment:
Does the required fail due to the systemd_reload action?

Reply:
Yeah essentially, because we are kinda lazy, we pack multiple resources within the same resource and not all the resources need the same parameter.
This was not a problem in chef 14 since the `:required` was not enforced. But now it is.